packages/node/tsconfig.json

```diff
@@ -4,7 +4,7 @@
   "include": ["src/**/*"],
 
   "compilerOptions": {
-    "lib": ["es2018"],
+    "lib": ["es2018", "es2020.string"],
```

Comment:
`string.matchAll` and everything else in `es2020.string` is fully supported by all the Node versions we support